pkg/backup/util/remote.go

```diff
@@ -634,6 +637,16 @@ func newGcsStorageOptionForFlag(conf *gcsConfig, flag string) []string {
 func newAzblobStorageOptionForFlag(conf *azblobConfig, flag string) []string {
 	var azblobOptions []string
 	path := fmt.Sprintf("azure://%s/", path.Join(conf.container, conf.prefix))
+	values := url.Values{}
+	if conf.storageAccount != "" {
+		values.Add("account-name", conf.storageAccount)
+	}
+	if conf.sasToken != "" {
+		values.Add("sas-token", conf.sasToken)
+	}
+	if v := values.Encode(); v != "" {
+		path = path + "?" + v
```

Comment:
Ref: https://docs.pingcap.com/tidb/stable/backup-and-restore-storages#uri-examples